installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -171,18 +186,25 @@ EOF
     fi
     vagrant ssh <<EOF
 $(install_vars_export)
-curl $(install_depot_server)/index.sh | bash -s redhat-dev
+curl $(install_depot_server)/index.sh | \
+  bivio_home_env_ignore_git_dir_ownership=$(vagrant_dev_ignore_git_dir_ownership $os) \
+  bash -s redhat-dev
 EOF
     vagrant_dev_post_install
 }
 
 vagrant_dev_mounts() {
+    declare first="${1:+1}"
     if [[ ${vagrant_dev_no_mounts:+1} ]]; then
         echo 'config.vm.synced_folder ".", "/vagrant", disabled: true'
         return
     fi
-    # Have to use vers=3 b/c vagrant will insert it (incorrectly) otherwise. Not sure why.
-    declare f=' type: "nfs", mount_options: ["nolock", "fsc", "actimeo=2"], nfs_version: 3, nfs_udp: false'
+    declare d=false
```

Comment:
I particularly dislike this construct, but there's nothing really better. People try to write [some pretty crazy stuff](https://stackoverflow.com/q/3953645), but in the end, this is probably the most straight forward. However, I just thought of this one, which I haven't seen used elsewhere:

```sh
declare d=${first:+true}
: ${d:=false}
```

Reply:
I don't have a strong opinion either way